### ISSUE TITLE
Player not in line of sight when next to a wall.

### ISSUE THREAD

**Issue by Hozu** [DarkstarProject/darkstar#2835]
Client Version (type /ver in game) :
30160203_0

Server Version (type @revision in game) :
14a1e3d

Source Branch (master/stable) :
master

Additional Information (Steps to reproduce/Expected behavior) :
http://i.imgur.com/G3IEiGU.png Also when casting on self there shouldn't be a line of sight check, heh.

**Comment by helixhamin** [comment]
I encountered the self cast line of sight issue yesterday when trying to cast protectra...

**Comment by teschnei** [comment]
I fixed self-cast line of sight.  Since it didn't have its own issue, I didn't close one for it

**Comment by helixhamin** [comment]
Hmm, I had the self cast with https://github.com/DarkstarProject/darkstar/commit/14a1e3d2f81547767a4157fb8d9762a9ff3b99e6

**Comment by Hozu** [comment]
Well yeah that commit is before https://github.com/DarkstarProject/darkstar/commit/0457ada8069ce14bca50ad2c718e18531544d0b6 - try it on that one or a later one.

**Comment by helixhamin** [comment]
I still have multiple issues of this, even with 3/6 builds.
I am beginning to wonder if I should just remove my navmeshes from my server for now, because it is rather frustrating that sometimes I cannot pull a mob when I am obviously looking right at it with an unobstructed view. Instead, I am forced to just go up to it and start hitting, which plays hell in areas where I need to be careful with pulls...

**Comment by teschnei** [comment]
considering the issue is open still...

**Comment by helixhamin** [comment]
Exactly. That is why I am wondering: Will removing the navmeshes also remove this issue until a resolution is reached?

**Comment by teschnei** [comment]
Yep.
